metabase/metabase
# Consistent "Verify" button for models

Currently, models use the same "Verify" button as regular saved questions. It's supposed to look a bit different to match the rest of the model management actions (like "Edit query" and "Customize metadata"). This PR fixes that

### To Verify

1. Spin up Metabase with BUCM (bottom-up content moderation) paid feature enabled
2. Open any model, click its title to open the details sidebar
3. You should see the "Verify this model" button matching the look of other model actions
4. Click it, it should be replaced with a "You verified this" banner with a small top margin. The "verified" icon should appear next to the model's name in the QB header
5. Hover the verification banner and click the "close" icon. The banner should be replaced with the "Verify this model" button and the "verified" icon should be removed from the model name
6. Open a saved question and repeat the same steps, the button should look differently

### Demo

**Before**

<img width="345" alt="CleanShot 2022-01-20 at 17 22 15@2x" src="https://user-images.githubusercontent.com/17258145/150372316-fcf970b4-32ba-4c16-9b22-10d9a93141b9.png">

**After**

![CleanShot 2022-01-20 at 17 34 31](https://user-images.githubusercontent.com/17258145/150372338-860ac520-60c8-4ebe-93fe-9d14ca00ae3c.gif)
